.ci/updatecli.d/update-json-specs.yml

```diff
@@ -0,0 +1,120 @@
+name: update-jsons-specs
+pipelineid: update-json-specs
+
+scms:
+  default:
+    kind: github
+    spec:
+      user: '{{ requiredEnv "GIT_USER" }}'
+      email: '{{ requiredEnv "GIT_EMAIL" }}'
+      owner: elastic
+      repository: apm-agent-nodejs
+      token: '{{ requiredEnv "GITHUB_TOKEN" }}'
+      username: '{{ requiredEnv "GIT_USER" }}'
+      branch: main
+
+sources:
+  sha:
+    kind: file
+    spec:
+      file: 'https://github.com/elastic/apm/commit/main.patch'
+      matchpattern: "^From\\s([0-9a-f]{40})\\s"
+    transformers:
+      - findsubmatch:
+          pattern: "[0-9a-f]{40}"
+
+  cgroup_parsing.json:
+    kind: file
+    spec:
+      file: https://raw.githubusercontent.com/elastic/apm/main/tests/agents/json-specs/cgroup_parsing.json
+  service_resource_inference.json:
+    kind: file
+    spec:
+      file: https://raw.githubusercontent.com/elastic/apm/main/tests/agents/json-specs/service_resource_inference.json
+  span_types.json:
+    kind: file
+    spec:
+      file: https://raw.githubusercontent.com/elastic/apm/main/tests/agents/json-specs/span_types.json
```

Comment:
It sure would be nice if updatecli supported a `kind: git-repo-subdir` source or something like that. :)

Reply:
That would be nice, yeah. 😅

Reply:
https://github.com/updatecli/updatecli/issues/1218 is the feature request I just created, please feel free to add further details if I missed something

Reply:
I've received some feedback about how to implement this particular feature using the existing capabilities, https://github.com/updatecli/updatecli/issues/1218#issuecomment-1463764985

what do you think?

Reply:
Looking at the suggested options, there seems to be no way around specifying every single file.